osu.Game/GameModes/Play/PlaySongSelect.cs

```diff
@@ -278,8 +295,10 @@ private void addBeatmapSet(BeatmapSetInfo beatmapSet)
 
         private void addBeatmapSets()
         {
-            foreach (var beatmapSet in database.Query<BeatmapSetInfo>())
-                addBeatmapSet(beatmapSet);
+            if (database.Query<BeatmapSetInfo>().Count() > 0)
+                foreach (var beatmapSet in database.Query<BeatmapSetInfo>())
+                    addBeatmapSet(beatmapSet);
+            else playButton.Hide();

```

Comment:
As far as I can see, there is no way to restore playButton's visibility. I guess .Show() should be called somewhere(for example, in the if block above).